lyse/communication.py

```diff
@@ -0,0 +1,125 @@
+#####################################################################
+#                                                                   #
+# /main.py                                                          #
+#                                                                   #
+# Copyright 2013, Monash University                                 #
+#                                                                   #
+# This file is part of the program lyse, in the labscript suite     #
+# (see http://labscriptsuite.org), and is licensed under the        #
+# Simplified BSD License. See the license.txt file in the root of   #
+# the project for the full license.                                 #
+#                                                                   #
+#####################################################################
+
+"""
+Code required for interprocess communication
+"""
+
+# 3rd party imports:
+import numpy as np
+
+# Labscript imports
+from labscript_utils.ls_zprocess import ZMQServer
+import labscript_utils.shared_drive as shared_drive
+
+# qt imports
+from qtutils import inmain_decorator
+
+# Lyse imports
+from lyse import _rangeindex_to_multiindex
```

Comment:
Since this is generally a refactoring point there is not reason not to do this now, so I did and also renamed it to be just `rangeindex_to_multiindex` since there is no reason to have an underscore if it is stored in `dataframe_utilities`.

Reply:
@ispielma This change broke the package on import. Needs to be either `from .dataframe_utilities import ...` or matching the style we have been using `from lyse.dataframe_utilities import ...`